### PR TITLE
DR 26B Alignment SelectionPage (merged with develop)

### PR DESCRIPTION
**Pull request by cheesehero112**
### Summary of Changes

This branch has the same changes but merged with develop after @RubySpeeders 's last PR. 

- Picture size to 150px (slightly smaller) to match Figma
- Buttons and return links are now on lower right
- “Return to -” links are white and underlined

---

### Summary of Implementation Logic

- Created new stylings for Return links and Button container 

---

### Issue Link

Original Issue: [Task 26](https://www.notion.so/Alignment-of-items-on-Selection-Page-05af333a23e44777a66e690804d0676f?pvs=4)

---

### Screen shots / Videos

Screenshot 2024-02-14 at 11 25 06 PM
Screenshot 2024-02-14 at 11 25 20 PM

---

### Acceptance Criteria

The goal of this ticket is to match the Figma design of the Selection Page, so it is not technically done yet, but I feel that these changes will improve the design a lot. We might want to create smaller tickets to continue the effort of implementing Figma design.  
---

### Checklist for PR Owner

- [x] Standardized PR Title
- [x] Included relevant videos and screen shots
- [x] Included Summary of Changes
- [x] Included Summary of Implementation Logic
- [ ] Acceptance Criteria are fufilled

### CODE REVIEW

File: client/src/app/assignments/page.tsx [hunk 1]
```diff
@@ -26,7 +62,18 @@ export default function Assignments() {
...
+          <div
+            key={assignment.musician.musician_id}
+            className={classes.container}
+          >
+            <ItemCard item={assignment.musician} />
+            <ItemCard item={assignment.instrument} />
+          </div>
+        );
+      })}
+      <div className={classNames(classes.buttonContainer)}>
```
Comment (RubySpeeders) [inline client/src/app/assignments/page.tsx:76]: I think we actually don't need to wrap this with classNames? I believe we really only need to use it if we are trying to use more than one class, or if dealign with conditional styling.

File: client/src/app/assignments/page.tsx [hunk 2]
```diff
@@ -36,21 +83,20 @@ export default function Assignments() {
...
           type={"button"}
+          style={{
+            borderRadius: "3.75em",
+            fontSize: "1rem",
+            padding: ".75em 4.75em",
+          }}
         >
           Return to Selection Page
         </Button>
+        <div className={classNames(classes.returnLink)}>
```
Comment (RubySpeeders) [inline client/src/app/assignments/page.tsx:94]: same as comment above.

File: client/src/components/SelectionContainer.tsx [hunk 3]
```diff
@@ -140,12 +165,14 @@ export default function SelectionContainer({
         </Box>
         {/* only render link tag if selection criteria are met */}
         {!isSelected || !selectedEqual ? (
-          <CustomButton
-            variant="contained"
-            disabled={!isSelected || !selectedEqual}
-          >
-            Assign
-          </CustomButton>
+          <div className={classNames(classes.buttonContainer)}>
```
Comment (RubySpeeders) [inline client/src/components/SelectionContainer.tsx:168]: same here

File: client/src/components/SelectionContainer.tsx [hunk 4]
```diff
@@ -140,12 +165,14 @@ export default function SelectionContainer({
...
+            <CustomButton
```
Comment (RubySpeeders) [inline client/src/components/SelectionContainer.tsx:169]: this button is not the same as on the assignments page. Is it possible to match it?

File: client/src/components/SelectionContainer.tsx [hunk 5]
```diff
@@ -155,17 +182,23 @@ export default function SelectionContainer({
               },
             }}
           >
-            <CustomButton
-              variant="contained"
-              disabled={!isSelected || !selectedEqual}
-            >
-              Assign
-            </CustomButton>
+            <div className={classNames(classes.buttonContainer)}>
```
Comment (RubySpeeders) [inline client/src/components/SelectionContainer.tsx:185]: same here

File: client/src/components/SelectionContainer.tsx [hunk 6]
```diff
@@ -155,17 +182,23 @@ export default function SelectionContainer({
...
+              >
+                Assign
+              </CustomButton>
+            </div>
           </Link>
         )}
-        <Link href="/">
-          <Typography>Return to homepage</Typography>
-        </Link>
+        <div className={classNames(classes.buttonContainer)}>
```
Comment (RubySpeeders) [inline client/src/components/SelectionContainer.tsx:195]: same for both of these!

**Review by RubySpeeders — CHANGES_REQUESTED** [review]
I think overall it's looking so much better! Excited to see these changes. If we can get the button on selection page looking right, I think that will really change the look of the app.

Also, do you mind making the item cards left aligned? I think changing the `justify-content` from `space-evenly` to `flex-start`. Thank you!

File: client/src/app/assignments/page.tsx (hunk 1 above)
Comment (cheesehero112) [reply client/src/app/assignments/page.tsx:76]: Got it! I'll go through them and get rid of the wrapper.

File: client/src/components/SelectionContainer.tsx (hunk 4 above)
Comment (cheesehero112) [reply client/src/components/SelectionContainer.tsx:169]: I was able to match it by editing `<CustomButton  />` and using inline styling instead. Not sure why the useStyles() weren't working correctly in this button.

**Comment by cheesehero112** [comment]
@RubySpeeders I fixed everything, except I'm not sure what the end result should look like with aligning `<ItemCards />` to left.
When I put `flex-start` to musicians makeStyles() function in SelectionContainer.tsx, the h2 titles (Select Musicians & Select Instruments) don't align with images as you change the window size. (see screenshot) 

I see that our Figma design has the h2 titles aligned left, but the musicians & instruments images seem to be in an even grid. Perhaps this should be a different ticket if the goal is to match the figma design (minus the drop down filter menu).

Screenshot 2024-02-29 at 9 25 56 PM

**Comment by cheesehero112** [comment]
@RubySpeeders My last commit has all the changes requested including adding the "flex-start", but I forgot to merge with the recent develop. Looks like there shouldn't be any merge conflict, but just wanted to mention that here.

**Comment by RubySpeeders** [comment]
changes look really good! I think maybe `align-items` can be `flex-start` instead of center, which will get the `h2` also left-aligned, and then some padding around the whole area so that it doesn't look so drastic? We could move this to another ticket if it makes you more comfortable!

I just merged with develop, and as you suspected, no merge conflicts :)

**Comment by cheesehero112** [comment]
@RubySpeeders Ok - let me know what you think. When I pulled develop, the font of h2 somehow became different, so I used <Typography> and inline styling to match Figma font size and style. (I tried to first use makeStyles() but it didn't work)

**Review by RubySpeeders — APPROVED** [review]
I think this looks great, and is definitely a big step forward in matching figma designs. I think maybe there is a way for us to clean up the styling a bit, but I think that will come when we set up a big style config section. Maybe that can be a Q2 or Q3 goal!

**Pull request merged**